worker/src/main/java/gov/cms/ab2d/worker/HealthAPI.java

```diff
@@ -40,7 +43,12 @@
     @GetMapping(VALIDATE_BFD_ENDPOINT)
     public ResponseEntity<Void> isBfdUp() {
         try {
-            bfdClient.capabilityStatement();
+            bfdClient.capabilityStatement(Versions.FhirVersions.STU3);
```

Comment:
If this ever fails don't you want to send a message specific to STU3?

Reply:
Also will there ever be a case where one api version is down and the other is up, both are hosted on the same Java Runtime.

Reply:
Sure, in theory.